TODO/things-i-wish-i-knew-before-i-wrote-my-first-android-app.md

```diff
@@ -41,15 +41,14 @@ If you’re an experienced Java developer, items 1, 2, and 5 probably won’t b
         }
     }
 
-This might seem like an impossible mistake for anyone to make. Its not. I made this mistake. I’ve seen others make this mistake, and I’ve interviewed people who weren’t very quick at figuring out why this is a mistake in the first place. Don’t do this. Its a n00b move.
 
-If MeTrackerStore keeps a reference to the Activity passed into its constructor, the Activity will never be garbage collected. (Unless the static variable is reassigned to a different Activity.) This is because mMeTrackerStore is static, and the memory for static variables is allocated when the application first starts and isn’t collected until the process in which the application is running quits.
+这事儿看起来也许是二的不一般的人才会做的事情。 但是事实上－ －我看过好多人犯这个错 并且我和这些不能在第一时间识别出这些错误的人交流过。 绝壁不要这么做－ －这是非常傻叉的行为
 
-If you find yourself tempted to do this, there’s probably something seriously wrong with your code. Find help. Maybe looking at Google’s Udacity course on [“Android Development for Beginners”](https://www.udacity.com/course/android-development-for-beginners--ud837) will help you out.
+如果当这个Activity传递到它的构造函数里，MeTrackerStore持有这个Activity的引用，那这个Activity将仍然不会被回收（除非这个静态变量被重新赋值给另一个Activity）。这是因为mMeTrackerStore是静态的，静态变量的内存是不会被回收的，直到程序里正在运行的进程停止。如果你发现自己尝试这么做，那么你的代码可能有一些严重的错误。寻找帮助的话，可以看看Google’s Udacity里的课程 [“Android Development for Beginners”](https://www.udacity.com/course/android-development-for-beginners--ud837)。
```

Comment:
**This is because mMeTrackerStore is static, and the memory for static variables is allocated when the application first starts and isn’t collected until the process in which the application is running quits.**
> 这是因为mMeTrackerStore是静态的，静态变量的内存是不会被回收的，直到程序里正在运行的进程停止。

建议翻译：这是因为mMeTrackerStore是静态的，静态变量的内存是应用程序开始启动就分配的，而且一直不会被回收直到应用程序的进程退出